### YamlMime:ManagedReference
items:
- uid: System.Workflow.Activities.Rules.RuleEvaluationException
  id: RuleEvaluationException
  children:
  - System.Workflow.Activities.Rules.RuleEvaluationException.#ctor
  - System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String)
  - System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  - System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: RuleEvaluationException
  nameWithType: RuleEvaluationException
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException
  type: Class
  summary: "Representa a classe base para todas as exceções causado por problemas de avaliação da regra."
  syntax:
    content: 'public class RuleEvaluationException : System.Workflow.Activities.Rules.RuleException'
  inheritance:
  - System.Exception
  - System.Workflow.Activities.Rules.RuleException
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor
  id: '#ctor'
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  langs:
  - csharp
  name: RuleEvaluationException()
  nameWithType: RuleEvaluationException.RuleEvaluationException()
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException()
  type: Constructor
  assemblies:
  - System.Workflow.Activities
  namespace: System.Workflow.Activities.Rules
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Workflow.Activities.Rules.RuleEvaluationException&quot;> </xref> classe."
  remarks: "Este construtor inicializa a <xref:System.Exception.Message%2A>propriedade da instância do novo a uma mensagem fornecido pelo sistema que descreve o erro e tem em consideração o idioma do sistema atual.</xref:System.Exception.Message%2A>       A tabela seguinte mostra os valores de propriedade inicial para uma instância do <xref:System.Workflow.Activities.Rules.RuleEvaluationException>.</xref:System.Workflow.Activities.Rules.RuleEvaluationException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Uma referência nula (`Nothing` no Visual Basic). |   | <xref:System.Exception.Message%2A>| Descrição localizada de um sistema fornecido. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuleEvaluationException ();
    parameters: []
  overload: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  langs:
  - csharp
  name: RuleEvaluationException(String)
  nameWithType: RuleEvaluationException.RuleEvaluationException(String)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(String)
  type: Constructor
  assemblies:
  - System.Workflow.Activities
  namespace: System.Workflow.Activities.Rules
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Workflow.Activities.Rules.RuleEvaluationException&quot;> </xref> classe com uma mensagem de erro especificado."
  remarks: "Este construtor inicializa a <xref:System.Exception.Message%2A>propriedade da nova instância utilizando o `message` parâmetro.</xref:System.Exception.Message%2A>       A tabela seguinte mostra os valores de propriedade inicial para uma instância do <xref:System.Workflow.Activities.Rules.RuleEvaluationException>.</xref:System.Workflow.Activities.Rules.RuleEvaluationException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| Uma referência nula (`Nothing` no Visual Basic). |   | <xref:System.Exception.Message%2A>| A cadeia de mensagem de erro. |</xref:System.Exception.Message%2A> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuleEvaluationException (string message);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem que descreve o erro."
  overload: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  langs:
  - csharp
  name: RuleEvaluationException(SerializationInfo,StreamingContext)
  nameWithType: RuleEvaluationException.RuleEvaluationException(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(SerializationInfo,StreamingContext)
  type: Constructor
  assemblies:
  - System.Workflow.Activities
  namespace: System.Workflow.Activities.Rules
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Workflow.Activities.Rules.RuleEvaluationException&quot;> </xref> classe com dados serializados."
  remarks: "Este construtor é chamado durante a anulação da serialização para reconstitute o objeto de exceção transmitido através de uma transmissão em fluxo."
  syntax:
    content: protected RuleEvaluationException (System.Runtime.Serialization.SerializationInfo serializeInfo, System.Runtime.Serialization.StreamingContext context);
    parameters:
    - id: serializeInfo
      type: System.Runtime.Serialization.SerializationInfo
      description: "O <xref href=&quot;System.Runtime.Serialization.SerializationInfo&quot;> </xref> que contém os dados de objecto serializado sobre a exceção que está a ser emitida."
    - id: context
      type: System.Runtime.Serialization.StreamingContext
      description: "O <xref:System.Runtime.Serialization.StreamingContext>que contém a nível contextual das informações sobre a origem ou de destino.</xref:System.Runtime.Serialization.StreamingContext>"
  overload: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  langs:
  - csharp
  name: RuleEvaluationException(String,Exception)
  nameWithType: RuleEvaluationException.RuleEvaluationException(String,Exception)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(String,Exception)
  type: Constructor
  assemblies:
  - System.Workflow.Activities
  namespace: System.Workflow.Activities.Rules
  summary: "Inicializa uma nova instância do <xref href=&quot;System.Workflow.Activities.Rules.RuleEvaluationException&quot;> </xref> classe, com uma mensagem de erro especificado e uma referência a interna <xref:System.Exception>que é a causa da <xref:System.Exception>.</xref:System.Exception> </xref:System.Exception>"
  remarks: "Um <xref:System.Exception>que é acionada como resultado direto de anterior <xref:System.Exception>deve incluir uma referência a exceção anterior no <xref:System.Exception.InnerException%2A>propriedade.</xref:System.Exception.InnerException%2A> </xref:System.Exception> </xref:System.Exception> O <xref:System.Exception.InnerException%2A>propriedade devolve o mesmo valor que é transmitido no construtor ou uma referência nula (`Nothing`) se o <xref:System.Exception.InnerException%2A>propriedade não fornece a interna <xref:System.Exception>valor para o construtor.</xref:System.Exception> </xref:System.Exception.InnerException%2A> </xref:System.Exception.InnerException%2A>       A tabela seguinte mostra os valores de propriedade inicial para uma instância do <xref:System.Workflow.Activities.Rules.RuleEvaluationException>.</xref:System.Workflow.Activities.Rules.RuleEvaluationException>      | Propriedade | Valor |   |--------------|-----------|   | <xref:System.Exception.InnerException%2A>| A referência de <xref:System.Exception>interna. |   | <xref:System.Exception.Message%2A>| A cadeia de mensagem de erro. |</xref:System.Exception.Message%2A> </xref:System.Exception> </xref:System.Exception.InnerException%2A>"
  syntax:
    content: public RuleEvaluationException (string message, Exception ex);
    parameters:
    - id: message
      type: System.String
      description: "A mensagem que descreve o erro."
    - id: ex
      type: System.Exception
      description: "O <xref:System.Exception>que é a causa da atual <xref:System.Exception>.</xref:System.Exception> </xref:System.Exception> Se o `innerException` parâmetro não é uma referência nula (<xref uid=&quot;langword_csharp_Nothing&quot; name=&quot;Nothing&quot; href=&quot;&quot;> </xref> no Visual Basic), o atual <xref:System.Exception>é gerado num bloco catch que processa a interna <xref:System.Exception>.</xref:System.Exception> </xref:System.Exception>"
  overload: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Workflow.Activities.Rules.RuleException
  isExternal: false
  name: System.Workflow.Activities.Rules.RuleException
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException()
  nameWithType: RuleEvaluationException.RuleEvaluationException()
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException()
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String)
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException(String)
  nameWithType: RuleEvaluationException.RuleEvaluationException(String)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException(SerializationInfo,StreamingContext)
  nameWithType: RuleEvaluationException.RuleEvaluationException(SerializationInfo,StreamingContext)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(SerializationInfo,StreamingContext)
- uid: System.Runtime.Serialization.SerializationInfo
  parent: System.Runtime.Serialization
  isExternal: false
  name: SerializationInfo
  nameWithType: SerializationInfo
  fullName: System.Runtime.Serialization.SerializationInfo
- uid: System.Runtime.Serialization.StreamingContext
  parent: System.Runtime.Serialization
  isExternal: true
  name: StreamingContext
  nameWithType: StreamingContext
  fullName: System.Runtime.Serialization.StreamingContext
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor(System.String,System.Exception)
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException(String,Exception)
  nameWithType: RuleEvaluationException.RuleEvaluationException(String,Exception)
  fullName: System.Workflow.Activities.Rules.RuleEvaluationException.RuleEvaluationException(String,Exception)
- uid: System.Exception
  parent: System
  isExternal: true
  name: Exception
  nameWithType: Exception
  fullName: System.Exception
- uid: System.Workflow.Activities.Rules.RuleEvaluationException.#ctor*
  parent: System.Workflow.Activities.Rules.RuleEvaluationException
  isExternal: false
  name: RuleEvaluationException
  nameWithType: RuleEvaluationException.RuleEvaluationException
